itemManagement/views.py

```diff
@@ -47,6 +48,20 @@ def get(self, request, itemId, *args, **kwargs):
             return render(request, 'itemManagement/item_details_assistant.html', context=context)
 
 
+class ItemCreationView(TemplateView):
+    def get(self, request, *args, **kwargs):
+        return render(request, 'itemManagement/item_creation.html')
+
+    def post(self, request, *args, **kwargs):
+        print("safas")
+        kghID = request.POST.get('item_id', "").strip()
```

Comment:
I'm a little confused, when is this post called? why does it only have the field kghID?